internal/cluster/sbd.go

```diff
@@ -86,23 +86,20 @@ func NewSBD(executor utils.CommandExecutor, cluster, sbdPath, sbdConfigPath stri
 	return s, nil
 }
 
-func getSBDConfig(sbdConfigPath string) (map[string]interface{}, error) {
-	sbdConfFile, err := os.Open(sbdConfigPath)
+func LoadSbdConfig(sbdConfigPath string) (map[string]string, error) {
+	sbdConfigFile, err := os.Open(sbdConfigPath)
 	if err != nil {
 		return nil, errors.Wrap(err, "could not open sbd config file")
 	}
 
-	defer sbdConfFile.Close()
-
-	sbdConfigRaw, err := io.ReadAll(sbdConfFile)
+	defer sbdConfigFile.Close()
```

Comment:
We should aim of checking the error output of the `Close` function.
At least to log it

Reply:
changed